web/pdf_find_controller.js

```diff
@@ -228,6 +245,7 @@ var PDFFindController = (function PDFFindControllerClosure() {
         this.resumePageIdx = null;
         this.pageMatches = [];
         var self = this;
+        var qwy = '';
```

Comment:
What's the purpose of this local var? Why not just set `self.state.query` directly?